### ISSUE TITLE
CacheWarmer issue with Propel

### ISSUE THREAD

**Issue by K-Phoen** [symfony2admingenerator/AdmingeneratorGeneratorBundle#242]
I have a little issue with the CacheWarmer and Propel. Before explaining it in details, I have to expose a bit the context in which it happens.
I'm currently working on automating my project's deployment process and I use capifony to achieve this. So when I do a deploy, the code in my repository is pulled/copied to the production server. All the code excepted the files generated by propel which are generated once the pull is complete.
And the problem is here. When capifony launches the command to generate my models, it appears that at some point the cache is "warmed up" by symfony and that the AdminGenerator CacheWarmer is called... but this class relies on propel models!
To summarize, the cache warmer needs my models, but to build them (from a fresh install/deploy) I need to warm the cache.

Do you have any clue to solve my issue ? Is there a way to build my models without triggering the cache warming, or perhaps the ability to disable the admin generator cache warmer, to trigger it by hand when needed?
